code/modules/mob/dead/observer/observer.dm

```diff
@@ -755,6 +755,20 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 	//	host.ckey = src.ckey
 	//	//host << "<span class='info'>You are now a mouse. Try to avoid interaction with players, and do not give hints away that you are more than a simple rodent.</span>"
 
+/mob/dead/observer/verb/find_arena()
+	set category = "Ghost"
+	set name = "Search For Arenas"
+	set desc = "Try to find an Arena to polish your robust bomb placement skills.."
+
+	if(!arenas.len)
+		usr << "There are no arenas in the world! Ask the admins to spawn one."
+		return
+
+	var/datum/bomberman_arena/arena_target = input("Which arena do you wish to reach?", "Arena Search Panel") in arenas

```

Comment:
I believe this is going to display a list of
>bomberman arena
>bomberman arena
>bomberman arena

also if you put as null|anything in arenas, it will give the input button a cancel button which is always nice